Adding a link to a Go reference implementation. (#205)
cfrg/draft-irtf-cfrg-hash-to-curve: README.md

This requires that you have the necessary software installed.  See
[the instructions](https://github.com/martinthomson/i-d-template/blob/master/doc/SETUP.md).

### Reference Implementations

The purpose of these implementations is for generating test vectors and enabling cross compatibility with other implementations.

> Note that these implementations cannot be used in production systems because side-channel countermeasures are not included.

Reviewer: ```suggestion
These implementations are for reference only. They MUST NOT be used in production systems.
```
